src/main/java/org/craftercms/studio/api/v2/dal/ClusterDAO.java

```diff
@@ -65,4 +65,11 @@
      * @return Cluster member with given id
      */
     ClusterMember getMemberById(long clusterMemberId);
+
+    /**
+     * Check if cluster member exists with given url
+     * @param memberUrl Member Url
+     * @return number of cluster members
```

Comment:
Do you mean the index of the cluster member?

Reply:
It is `id` field from cluster member model

Reply:
Please rename the parameter to indicate it's the `id` of the cluster member instead of `number`